components/victron/victron.cpp

```diff
@@ -471,6 +473,24 @@ void VictronComponent::handle_value_() {
     if ((device_type_text_sensor_ != nullptr) && !device_type_text_sensor_->has_state()) {
       device_type_text_sensor_->publish_state(device_type_text(value));
     }
+  } else if (label_ == "LOAD") {
+    // value = atoi(value_.c_str());
+
+    // ESP_LOGD(TAG, "received PID: '%s'", value_.c_str());
+    value = strtol(value_.c_str(), nullptr, 0);
+    // ESP_LOGD(TAG, "received PID: '%04x'", value);
+    if ((load_state_text_sensor_ != nullptr) && !load_state_text_sensor_->has_state()) {
+      load_state_text_sensor_->publish_state(value);
```

Comment:
Could you provide some example values? Are you sure a `text_sensor` is a good choice here? Same applies to the relay state. If it's a boolean let's expose a `binary_sensor`.